movienager/accounts/models.py

```diff
@@ -26,7 +26,7 @@ class User(AbstractUser):
     first_name = models.CharField(_("first name"), max_length=20, blank=True)
```

Comment:
do we need all of these translations in the model - `_()` calls look quite reduntant in our app?